### DISCUSSION TITLE
Can I run multiple instances of NPM in parallel ?

### DISCUSSION THREAD

**Discussion by felix-exon** [NginxProxyManager/nginx-proxy-manager#2919] [Q&A]
I am using different services in a docker swarm (3 nodes) and was wondering if I could run NPM on each node, all sharing the same db and files (basically the `/data` mount will be the same).

Is there any file-locking or will there be issues when running (like 3) NPM at the same time.

If possible I could then "load-balance" between the 3 proxies.

**Comment by iBerlioz** [comment]
@jc21 ?
